readme.md

```diff
@@ -58,81 +58,68 @@ const terminalImage = require('terminal-image');
 
 ## API
 
-Supports PNG and JPEG images.
+Supports PNG and JPEG images. Animated GIFs are also supported with `.gifBuffer` and `.gifFile`.
 
 ### terminalImage.buffer(imageBuffer, options?)
+### terminalImage.file(filePath, options?)
+### terminalImage.gifBuffer(imageBuffer, options?)
+### terminalImage.gifFile(filePath, options?)
 
 Returns a `Promise<string>` with the ansi escape codes to display the image.
 
-##### options
+#### options
 
 Type: `object`
 
-###### height
+##### height
 
 Type: `string | number`
 
 Custom image height.
 
 Can be set as percentage or number of rows of the terminal. It is recommended to use the percentage options.
 
-###### width
+##### width
 
 Type: `string | number`
 
 Custom image width.
 
 Can be set as percentage or number of columns of the terminal. It is recommended to use the percentage options.
 
-###### preserveAspectRatio
+##### preserveAspectRatio
 
 Type: `boolean`\
 Default: `true`
 
 Whether to maintain image aspect ratio or not.
 
-#### imageBuffer
-
-Type: `Buffer`
-
-Buffer with the image.
-
-### terminalImage.file(filePath, options?)
-
-Returns a `Promise<string>` with the ansi escape codes to display the image.
-
-#### filePath
-
-Type: `string`
-
-File path to the image.
-
-##### options
+##### maximumFrameRate
 
-Type: `object`
+**Only works for `terminalImage.gifBuffer` or `terminalImage.gifFile`**
 
-###### height
+Type: `number | Infinity`\
+Default: `60`
 
-Type: `string | number`
+Maximum framerate to render the GIF. This option is ignored when using iTerm.
 
-Custom image height.
+##### renderFrame
```

Comment:
This needs to describe why one might want to set this option.

Reply:
Fixed!